File: src/tests/DataBlock Sample Application/VB.NET/Regions/RegionPersistentObject.vb
Imports System
Imports System.Data
Imports voidsoft.DataBlock
Namespace ExtenderRegion

    Public Class RegionPersistentObject
        Inherits voidsoft.DataBlock.PersistentObject

        Public Sub New(ByVal database As EDatabase, ByVal connectionString As String, ByVal mainTable As TableMetadata)
            MyBase.New(database, connectionString, mainTable)
        End Sub

        Public Sub New(ByVal session As Session, ByVal mainTable As TableMetadata)
            MyBase.New(session, mainTable)
        End Sub
    End Class
End Namespace

File: src/tests/Console.Tests/RegionPersistentObject.cs
using System;
using System.Data;
using voidsoft.DataBlock;
namespace ExtenderRegion
{
public class RegionPersistentObject : voidsoft.DataBlock.PersistentObject
{ 
 
      public RegionPersistentObject(EDatabase database, string connectionString, TableMetadata mainTable) : base (database, connectionString, mainTable) 
      {
      }

    public RegionPersistentObject(Session s , TableMetadata mainTable) : base (s,mainTable)
    {
    }

}
}
